frontend/src/app/app-routing.module.ts

```diff
@@ -270,22 +274,21 @@ const routes: Routes = [
       },
       {
         path: 'sessions',
```

Comment:
You capitalized most of the breadcrumbs, but not all (for example, `sessions` and `overview`) are still lower case.

Reply:
Adjusted the sessions breadcrumb (in line 54) but the overview breadcrumb was already `Session Overview`. This here is only the path which can not be adjusted.